tests/election.py

```diff
@@ -35,6 +35,19 @@ def test_kill_primary_no_reqs(network, args):
             r = c.get("/node/network")
             c.wait_for_commit(r)
 
+            # Also verify that reported last ack time are as expected
```

Comment:
When a new primary node has been elected, we check that the backup nodes report `0` as the time of latest backup ACK (backups don't exchange append entries between each other!) and the primary reports sensible ACK times for each remaining backups.